student.rb

```diff
@@ -1,4 +1,4 @@
-require 'Person'
+require './Person'
 
 class Student < Person
   attr_reader :classroom
```

Comment:
- [ ] I encountered the error below when trying to create a student

![image](https://user-images.githubusercontent.com/16058470/136662205-a0638dda-2f3f-4071-b911-46bb6bdeba4b.png)

Kindly fix it.